plugin/mlsdbprovider.h

```diff
@@ -163,6 +163,9 @@ private Q_SLOTS:
     QBasicTimer m_idleTimer;    // qApp->quit() if positioning is off for long enough.
     QBasicTimer m_fixLostTimer; // after fix timeout, status set to Acquiring.  timer is reset when a position is calculated.
     QBasicTimer m_recalculatePositionTimer;
+
+    bool m_mlsFallbacksLacfEnabled;
+    bool m_mlsFallbacksIpfEnabled;
```

Comment:
`m_fallbacksLacf/m_mlsFallbacksLacfEnabled` looks to me like a single setting with 3 distinct states - `on`, `off` and `disabled`. I don't see any reason to split this one thing into two bits, introduce the fourth state (two bits = four states) and define these two bits in two different classes. Can you explain these manipulations?

The same applies to `m_fallbacksIpf/m_mlsFallbacksIpfEnabled` pair.

Reply:
I would represent this kind of thing as a `enum { On, Off, Disabled }` or the like.

Reply:
m_fallbacksIpf/m_mlsFallbacksIpfEnabled - Thanks, I'll change. 
We will use the state "disabled"? For what purposes do we need it?

Reply:
`m_mlsFallbacksIpfEnabled = true, m_fallbacksIpf ignored` => `Disabled`,
`m_mlsFallbacksIpfEnabled = true, m_fallbacksIpf = true` => `On`,
`m_mlsFallbacksIpfEnabled = true, m_fallbacksIpf = false` => `Off`
Those are the 3 states that make sense, right?

Reply:
By default, m_mlsFallbacksIpfEnabled = true is for someone who does not want to use the configuration file. There are only two states: On and Off. Why the third state?

Reply:
Sorry, I meant this:
`m_mlsFallbacksIpfEnabled = false, m_fallbacksIpf ignored => Disabled`
`m_mlsFallbacksIpfEnabled = true, m_fallbacksIpf = true => On`
`m_mlsFallbacksIpfEnabled = true, m_fallbacksIpf = false => Off`